iOS_SDK/OneSignalSDK/Source/OneSignalHelper.m

```diff
@@ -469,15 +471,15 @@ + (BOOL)handleIAMPreview:(OSNotificationPayload *)payload {
     return NO;
 }
 
-+(NSNumber*)getNetType {
++ (NSNumber*)getNetType {
     OneSignalReachability* reachability = [OneSignalReachability reachabilityForInternetConnection];
     NetworkStatus status = [reachability currentReachabilityStatus];
     if (status == ReachableViaWiFi)
         return @0;
     return @1;
 }
 
-+ (NSString *)getCurrentDeviceVersion {
++ (NSString*)getCurrentDeviceVersion {
```

Comment:
I think we are using the space between the type and the * by convention

Reply:
We have so many that have no space and so many that have a space. We should clean this up at some point haha. But got this one!